README.md

```diff
@@ -47,6 +47,7 @@
 - [LimeChat](http://limechat.net/mac/) - Instant messaging application. [![Open-Source Software][OSS Icon]](https://github.com/psychs/limechat)
 - [Messenger for Desktop](http://messengerfordesktop.com/) - An app for Facebook messenger. [![Open-Source Software][OSS Icon]](https://github.com/Aluxian/Facebook-Messenger-Desktop)
 - [Profanity](http://www.profanity.im) - A terminal XMPP client. [![Open-Source Software][OSS Icon]](https://github.com/boothj5/profanity)
+- [Slack](https://slack.com/apps) - A messaging app for teams.
```

Comment:
I was wondering if the link should point to App Store instead - `https://itunes.apple.com/in/app/slack/id803453959`